ui/components-react/src/components-react/newEditors/CommittingEditor.tsx

```diff
@@ -0,0 +1,67 @@
+/*---------------------------------------------------------------------------------------------
+ * Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+ * See LICENSE.md in the project root for license terms and full copyright notice.
+ *--------------------------------------------------------------------------------------------*/
+import * as React from "react";
+import { useRef, useState } from "react";
+import { Editor } from "./Editor.js";
+import type { Value } from "./values/Values.js";
+import type { EditorProps } from "./Types.js";
+
+type CommittingEditorProps = Omit<EditorProps, "onFinish" | "onChange"> & {
```

Comment:
This means, that `CommitingEditor` is not a "real editor" and can not be registered via `EditorsRegistryProvider`?

Reply:
Nvm, I see this is only adding some keyboard handlers for the regulard `Editor`.